core/templates/dev/head/domain/statistics/LearnerAnswerInfoObjectFactory.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating instances of frontend learner answer info
+ * domain objects.
+ */
+
+var oppia = require('AppInit.ts').module;
+
+oppia.factory('LearnerAnswerInfoObjectFactory', [
```

Comment:
is there any corresponding object in backend for this one? If yes, then why this doesn't have the interaction id (although it contains the answer)?

Reply:
Its a subset of another model LearnerAnswerDetailsModel which contains the interaction id. I will created LearnerAnswerDetails object factory in the 3rd MS.

Reply:
I see, so what's will be the basic diff in LearnerAnswerDetails and LearnerAnswerInfo? (The name seems a bit confusing to me.)

Reply:
LearnerAnswerDetails can contain a number of LearnerAnswerInfo objects.

Reply:
Done